README-permission.md

```diff
@@ -139,18 +139,18 @@ Variable | Description | Required
 `ipaadmin_principal` | The admin principal is a string and defaults to `admin` | no
 `ipaadmin_password` | The admin password is a string and is required if there is no admin ticket available on the node | no
 `name` \| `cn` | The permission name string. | yes
-`perm_rights` \| `ipapermright` | Rights to grant. It can be a list of one or more of `read`, `search`, `compare`, `write`, `add`, `delete`, and `all` default: `all` | no
+`right` \| `ipapermright` | Rights to grant. It can be a list of one or more of `read`, `search`, `compare`, `write`, `add`, `delete`, and `all` default: `all` | no
 `attrs` | All attributes to which the permission applies | no
-`perm_bind_rule_type` \| `ipapermbindruletype` | Bind rule type. It can be one of `permission`, `all`, `self`, or `anonymous` defaults to `permission` for new permissions.| no
-`perm_location` \| `ipapermlocation` | Subtree to apply permissions to | no
-`extra_target_filter` \| `extratargetfilter` | Extra target filter | no
-`perm_target_filter` \| `ipapermtargetfilter` | All target filters | no
-`perm_target` \| `ipapermtarget` | Optional DN to apply the permission to | no
-`perm_target_to` \| `ipapermtargetto` | Optional DN subtree where an entry can be moved to | no
-`perm_target_from` \| `ipapermtargetfrom` | Optional DN subtree from where an entry can be moved | no
-`member_of` \| `memberof` | Target members of a group (sets memberOf targetfilter) | no
-`target_group` \| `targetgroup` | User group to apply permissions to (sets target) | no
-`object_type` \| `type` | Type of IPA object (sets subtree and objectClass targetfilter) | no
+`bindtype` \| `ipapermbindruletype` | Bind rule type. It can be one of `permission`, `all`, `self`, or `anonymous` defaults to `permission` for new permissions.| no
+`subtree` \| `ipapermlocation` | Subtree to apply permissions to | no
+`filter` \| `extratargetfilter` | Extra target filter | no
+`rawfilter` \| `ipapermtargetfilter` | All target filters | no
+`target` \| `ipapermtarget` | Optional DN to apply the permission to | no
+`targetto` \| `ipapermtargetto` | Optional DN subtree where an entry can be moved to | no
+`targetfrom` \| `ipapermtargetfrom` | Optional DN subtree from where an entry can be moved | no
+`memberof` | Target members of a group (sets memberOf targetfilter) | no
+`targetgroup` | User group to apply permissions to (sets target) | no
+`type` | Type of IPA object (sets subtree and objectClass targetfilter) | no
```

Comment:
Please, add `object_type` as an alias  to `type`. The code already allows it, it is just missing in the documentation.

Reply:
So I actually have a question about this one. First I must admit this is the first python code I have done.

I generally avoid naming variables/fields 'type'. So the module is all object_type but he API is type. The code handles this currently by using type as an alias for object_type, but the docs are all written to type. If there is no reason for all of this then I will happily change it.

Reply:
I don't like `type` as a variable either, and we don't use it in the modules. You may use `permission_type` (similar to is used in ipavault's `vault_type`), or `object_type`, as you used, as this type is the permission object type. I prefer `object_type`.